src/classes/modules/java.base/jdk/internal/access/SharedSecrets.java

```diff
@@ -17,10 +17,9 @@
  */
 package jdk.internal.access;
 import jdk.internal.misc.Unsafe;
-import java.io.File;
-import java.io.FileDescriptor;
-import java.io.ObjectInputFilter;
-import java.io.ObjectInputStream;
+
+import java.io.*;
```

Comment:
Using wildcards is convenient but obscures which classes we actually import; please keep the explicit list of classes as before this change.